src/styles/blocks/main/content/_card.scss

```diff
@@ -0,0 +1,95 @@
+.card {
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: #fff;
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  transition: all 300ms;
+
+  &__content {
+    display: block;
+  }
+
+  &__image {
+    width: 100%;
+    height: 134px;
+    background-image: url('../images/imac.jpeg');
+    background-size: contain;
+    background-repeat: no-repeat;
+    margin-inline: auto;
+  }
+
+  &__title {
+    margin-top: 40px;
+    color: #060B35;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+
+  }
+
+  &__code {
+    margin-top: 4px;
+    color: $gray-color;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    justify-self: start;
+  }
+
+  &__reviewers {
+    display: flex;
+    width: 53px;
+    height: 16px;
+    flex-direction: column;
+    justify-content: flex-end;
```

Comment:
you could use `align-items` property with default flex-direction (row) which you don't need to set